CONTRIBUTING.md

```diff
@@ -1,80 +1,96 @@
 
-If a particular commit references another issue, please add the reference, for example "refs #1234", or "fixes #4321". Using "fixes or closes" keywords will cause the corresponding issue to be closed when the pull request is merged.
+- [Slack Channel](http://invite.bitcoinclassic.com/)
+- [Subreddit](https://www.reddit.com/r/bitcoin_classic)
+- [Consider.it](https://bitcoinclassic.consider.it/)
 
-Please refer to the [Git manual](https://git-scm.com/doc) for more information about Git.
+## <a name="merge-window"></a> Merge Window
+Using merge windows allows Bitcoin Classic to remain focused on specific issues at hand without distraction. A merge window decides the focus of development for the continuation of that window. In this window, code which is deemed to be sufficiently stable (and which is accepted by the community) is merged. Merge windows will be closed based on a vote via [consider.it](https://bitcoinclassic.consider.it/) and another vote will be held to open the next merge window. For details on the current merge window see below.
 
-  - Push changes to your fork
-  - Create pull request
+ - `Hard fork bitcoin to a 2 MB blocksize limit.`
+ - `2 MB Test Net`
+ - `Consensus Rule changes - O(n^2) protections`
+ - `Activication / versionbits code`
 
-The title of the pull request should be prefixed by the component or area that the pull request affects. Examples:
+Emergencies and bug fixes are outside the scope of merge windows.
 
-    Consensus: Add new opcode for BIP-XXXX OP_CHECKAWESOMESIG
-    Net: Automatically create hidden service, listen on Tor
-    Qt: Add feed bump button
-    Trivial: fix typo
+## <a name="considerit"></a> Consider.it
+By voting on issues, Bitcoin Classic aims to create a collective deliberation among all those using this implementation. Every vote must be respected by 3 groups: users, miners, and developers. **The votes are interpreted as such:**
 
-If a pull request is specifically not to be considered for merging (yet) please prefix the title with [WIP] or use [Tasks Lists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) in the body of the pull request to indicate tasks are pending.
+- Each vote goes from -1 to 1, with 0 at neutral/undecided
+- In the Miner vote, each Miner's vote is multiplied by their hashpower
+- Only verified users have votes.
 
-The body of the pull request should contain enough description about what the patch does together with any justification/reasoning. You should include references to any discussions (for example other tickets or mailing list discussions).
+As voting is a continuous process, all voters are encouraged to vote for a pull request only if they believe it has been thoroughly tested, reviewed, and fitting of any other criteria the voter holds fit. If any vote is found to be subject of vote manipulation the vote will be considered void.
```

Comment:
I added that line because it's possible the verification methods could have a flaw or at some point verified users could be willing to vote if their paid. Just a safety measure really.